Configurations/50-cppbuilder.conf

```diff
@@ -6,6 +6,7 @@ my %targets = (
         thread_scheme    => "winthreads",
         cc               => "bcc32c",
         CPP              => "cpp32 -oCON -Sc -Sr",
+        make_depend      => "cpp32 -oCON -Hp",
```

Comment:
@tanzislam, would you mind checking that this works as intended?  If not, this command should be expanded until it does.

Reply:
Looks like `-Sx` is also needed, otherwise the full output of the preprocessing is emitted. `-Sx` is [not actually documented](http://docwiki.embarcadero.com/RADStudio/Sydney/en/CPP32.EXE,_the_C_Compiler_Preprocessor) in the manual, but in the command-line help:

```
C:\Users\t_17_\source\repos\cryptopals\openssl>cpp32 -h -S
Embarcadero C++ Win32 Preprocessor 7.40 Copyright (c) 1993-2018 Embarcadero Technologies, Inc.
Available options (* = default setting, xxx = has sub-options: use -h -X):
(Note: -X- or -w-XXX will usually undo whatever was set or unset by -X or -wXXX.
 If two options conflict, the last one specified will be used.)
  -S      Control output format; see user documentation
  -Sc     Keep comments
  -Sd     Dump defines/undefs in preprocessed output
  -Sk     Keep output file on errors
  -Sr     Make output readable
  -Ss     Supply statistics
  -Sg     Emit info about header guards
  -Sl     Generate #line style output
  -SI     Accept IDL specific syntax
  -Sx     Omit preprocessed text in output
  -Si     Don't include files; leave include statements in output
  -Sm     Don't expand macros; leave macro definitions and calls in output
  -Sp     Don't preprocess; leave preprocessor conditionals in output
```

Reply:
Cool, thanks!